### PR TITLE
Navigate to sources page on nav click

### PR DESCRIPTION
**Pull request by karelhala**
JIRA: https://issues.redhat.com/browse/RHCLOUD-13096

### CODE REVIEW

**Comment by karelhala** [comment]
ping @rvsia

**Comment by Hyperkid123** [comment]
@rvsia there is a problem that we have two separate history instances. So We will include the fix to the chrome nav re-work, but we need this as a short-term solution.

**Comment by karelhala** [comment]
@rvsia not really, as chrome doesn't have access to the app state and we don't really know where to navigate user in the application. We don't want to issue app refresh as that is not user friendly and we've been sending the nav change for some time, it's just that we are sending the nav update also for active links.

File: src/App.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ const App = () => {
       console.warn('Failed to initialize chrome navigation.');
     }
 
+    insights.chrome.on('APP_NAVIGATION', (event) => event.navId === 'sources' && history.push(routes.sources));
```
Comment (rvsia) [inline src/App.js:31]: please use `routes.sources.path`

**Review by rvsia — COMMENTED** [review]
I think tests will fail, because insights.on does not exist.

Also, it would be nice to add a test case to check this behavior.

**Comment by rvsia** [comment]
@Hyperkid123 @karelhala
>  we need this as a short-term solution.

so are you going to update all the applications that are currently deployed?

> we don't really know where to navigate user in the application.

I think it's the same as the user clicks for the first time.

**Review by rvsia — APPROVED** [review]

**Pull request merged**